### ISSUE TITLE
ResNet101 Pytorch To IR error

### ISSUE THREAD

**Issue by chenghaz** [Microsoft/MMdnn#395]
Platform (like ubuntu 16.04/win10): ubuntu18.04

Python version:3.6

Source framework with version (like Tensorflow 1.4.1 with GPU):Pytorch 0.4.1

Destination framework with version (like CNTK 2.3 with GPU):IR

Pre-trained model path (webpath or webdisk path): mmdownload -f pytorch -n resnet101 -o ./

Running scripts: python3 -m mmdnn.conversion._script.convertToIR -f pytorch -d resnet101 --inputShape 3,224,224 -n imagenet_resnet101.pth

Error: 
  File ".../conversion/pytorch/pytorch_graph.py", line 74, in _optimize_graph
    graph = torch._C._jit_pass_onnx(graph, aten)
TypeError: _jit_pass_onnx(): incompatible function arguments. The following argument types are supported:
    1. (arg0: torch::jit::Graph, arg1: torch._C._onnx.OperatorExportTypes) -> torch::jit::Graph

**Comment by namizzz** [comment]
Hi @chenghaz ,MMdnn converts the pytorch model graph to onnx graph first then to IR. Maybe it's better to ask in [pytorch](https://github.com/pytorch/pytorch/issues)

**Comment by chenghaz** [comment]
@namizzz yep... They may change the function interface for converting to ONNX. Maybe we should change the second parameter of torch._C._jit_pass_onnx(graph, aten) to torch._C._jit_pass_onnxtorch._C._onnx.OperatorExportTypes.ONNX ?

**Comment by namizzz** [comment]
Hi @chenghaz , MMdnn only support PyTorch 0.4.0 now. There are a lot of errors in building graph if we update it to 0.4.1. We will update and fix it later. If you want to use MMdnn now, you can:
1. downgrade your pytorch to 0..4.0 
2. edit the code in https://github.com/Microsoft/MMdnn/blob/master/mmdnn/conversion/pytorch/pytorch_graph.py 
`graph = torch._C._jit_pass_onnx(graph, OperatorExportTypes.ONNX)`
and also get the right graph with correct `output_shape` line 142

Thanks! ping us if you need any further help!

**Comment by KeremTurgutlu** [comment]
Did support for v1.0 came? I am having trouble with converting from pytorch to IR. 

> TypeError: _jit_pass_onnx(): incompatible function arguments. The following argument types are supported:
    1. (arg0: torch::jit::Graph, arg1: torch._C._onnx.OperatorExportTypes) -> torch::jit::Graph

**Comment by SnowRipple** [comment]
The same here:

  File "/home/snow_ripple/workspace/012_mmdnn/mmdnn/conversion/pytorch/pytorch_graph.py", line 74, in _optimize_graph
    graph = torch._C._jit_pass_onnx(graph, aten)
TypeError: _jit_pass_onnx(): incompatible function arguments. The following argument types are supported:

    1. (arg0: torch::jit::Graph, arg1: torch._C._onnx.OperatorExportTypes) -> torch::jit::Graph

pytorch 1.1
